gdal/ogr/ogr_srs_panorama.cpp

```diff
@@ -436,6 +438,18 @@ OGRErr OGRSpatialReference::importFromPanorama( long iProjSys, long iDatum,
             oGCS.importFromEPSG( aoDatums[iDatum] );
             CopyGeogCSFrom( &oGCS );
         }
+        else if( iEllips == PAN_ELLIPSOID_GSK2011 )
+        {
+            OGRSpatialReference oGCS;
+            oGCS.importFromEPSG( 7683 );
+            CopyGeogCSFrom( &oGCS );
+        }
+        else if( iEllips == PAN_ELLIPSOID_PZ90 )
+        {
+            SetGeogCS( "PZ-90.11", "Parametry_Zemli_1990_11",
```

Comment:
Unfortunately, I can't find valid EPSG code for `PZ-90.11` 2D GeogCS .
Thus, I define GeogCS explicitly.